packages/types/lib/logs/messages.ts

```diff
@@ -43,10 +43,16 @@ export interface OperationAction {
     type: 'action';
     action: 'run';
 }
+
+export interface OperationOnEvents {
+    type: 'events';
+    action: 'run';
```

Comment:
why not put the actual event name here?

Reply:
at first I though that we wouldn't need to update the Operation when adding new event types but yes we could differentiate based on the event type. what do you prefer?